crypto/rand/rand_lcl.h

```diff
@@ -185,6 +185,11 @@ struct rand_drbg_st {
      */
     struct rand_pool_st *pool;
```

Comment:
BTW: would you like me to rename "pool" to maybe "seed_pool" or something?

Reply:
Yes, the asymmetry between "pool" and "adin_pool" disturbed me, too.  But I'd prefer `entropy_pool`, because it is used for the `get_entropy` callback.

Reply:
At it is important that the renaming happens both on master and 1.1.1  :-)

Reply:
Yeah for sure :)  I would like to do that in a follow-up PR.
Regarding the name, it should make it clear that this entropy_pool is only
used for seeding, thus used in get_entropy but only exceptional, when
called from drbg_add.  Would you still prefer entropy_pool over say add_pool ?

Reply:
Yeah, you are right. It's only for RAND_add() and RAND_seed(). Since add_pool and adin_pool sound too similar, then seed_pool is probably the better alternative.